src/components/blocks/wrappers/ConditionalChildrenLanguageDisplay.js

```diff
@@ -8,11 +8,11 @@ const ConditionalChildrenLanguageDisplay = ({ children }) => {
   let currentGroup = false;
   const childLanguageGroups = [];
   const toFilter = [];
-  Children.forEach(children, ({ props, props: { attribs = null } }, index) => {
```

Comment:
could also become TS, if you're in the mood for it 😅

Reply:
I've investigated and it causes a lot more issues; TypeScript doesn't support unions as big as would need to be created here.